pr2_plugs_configuration: 
  detect_outlet:
    joint_names: ['r_shoulder_pan_joint', 'r_shoulder_lift_joint', 'r_upper_arm_roll_joint', 'r_elbow_flex_joint', 'r_forearm_roll_joint', 'r_wrist_flex_joint', 'r_wrist_roll_joint']
    waypoints: 
    - [[-0.74410352784063805, 0.88607370182202605, -0.83666299097181296, -1.9971028134584377, 1.58,                -0.091021168910623257, 0.11380852814842102], [], [], 0.0]
    - [[-0.80,                0.80,                -0.90,                -0.8,                3.0,                 -0.08653975014036612,  0.16362624166245854], [], [], 0.0]
    - [[-1.10,                1.0,                 -0.60,                -0.4,                4.5,                 -0.08653975014036612,  0.16362624166245854], [], [], 0.0]
    - [[-1.363,               1.278,                0.276,               -0.4,                6.264,               -0.090,                0.0411             ], [], [], 0.0]
  recover_outlet_to_free:
    joint_names: ['r_shoulder_pan_joint', 'r_shoulder_lift_joint', 'r_upper_arm_roll_joint', 'r_elbow_flex_joint', 'r_forearm_roll_joint', 'r_wrist_flex_joint', 'r_wrist_roll_joint']
    waypoints:
    - [[-1.363,               1.278,                0.276,               -0.4,                6.264,               -0.090,                0.0411             ], [], [], 0.0]
    - [[-1.9236988795958316,  1.3,                 -0.25,                -0.65,               6.264,               -0.090,                0.0411             ], [], [], 0.0]
    - [[-1.2,                 1.3,                 -0.5,                 -1.5,                3.5,                 -0.090,                0.16362624166245854], [], [], 0.0]
    - [[-0.74410352784063805, 0.88607370182202605, -0.83666299097181296, -1.9971028134584377, 0.58824708437894957, -0.091021168910623257, 0.11380852814842102], [], [], 0.0]
